### PR TITLE
fixed startup script

### PR DESCRIPTION
**Pull request by rinslow**

### CODE REVIEW

File: rasp_init/create-service-file.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 WorkingDirectory={main_dir}
 Environment=DISPLAY={display}
 Environment=XAUTHORITY={xauthority}
-Restart=on-failure
+Restart=always
```
Comment (osherdp) [inline rasp_init/create-service-file.py]: Maybe on-failure is better
It might be annoying not being able to close the app with alt-F4

Comment (avrahamshukron) [inline rasp_init/create-service-file.py]: Why not only on failure?

Comment (rinslow) [reply rasp_init/create-service-file.py]: on failure doesn't work because we catch all errors and return code is 0

Comment (rinslow) [reply rasp_init/create-service-file.py]: it's a white room script, not Development

**Pull request merged**